public/locales/en/common.json

```diff
@@ -400,7 +403,10 @@
       "info": {
         "sources": "Kabbalah Sources",
         "topics": "Topics",
-        "collections": "Related to"
+        "collections": "Related to",
```

Comment:
Please make translations to all languages the same.

Reply:
added